validator/sawtooth_validator/journal/genesis.py

```diff
@@ -212,11 +217,17 @@ def _get_block_publisher(self, state_hash):
         """
         state_view = self._state_view_factory.create_view(state_hash)
         try:
+            class BatchPublisher(object):
+                def send(self, transactions):
+                    raise InvalidGenesisConsensusError(
```

Comment:
Agreed.  The system needs to be bootstrapped with the first validator.  And, as @cintel states, we cannot apply the C block policy until there are enough blocks in the blockchain.  It is going to have to be some factor of c and the number of validators as it also has to play nice with the K policy (i.e., a validator has to refresh signup information after claiing K blocks) and the z-Test, which prevents a validator from winning block elections too often.